controllers/release/adapter.go

```diff
@@ -90,15 +90,24 @@ func (a *adapter) EnsureConfigIsLoaded() (controller.OperationResult, error) {
 	}
 
 	var err error
+	a.logger.Info("Getting Release Service Config")
 	a.releaseServiceConfig, err = a.loader.GetReleaseServiceConfig(a.ctx, a.client, v1alpha1.ReleaseServiceConfigResourceName, namespace)
 	if err != nil && !errors.IsNotFound(err) {
 		return controller.RequeueWithError(err)
 	}
 
 	if err != nil {
+		a.logger.Info("Creating Empty Release Service Config")
 		a.releaseServiceConfig = a.getEmptyReleaseServiceConfig(namespace)
+		patch := client.MergeFrom(a.releaseServiceConfig.DeepCopy())
```

Comment:
There's no need to persist it. If we really want to persist it maybe that should be done in a different place (whenever the manager is created and not per reconciliation).

Reply:
We need to persist it. The release pipelines want to do a `oc get RSC/release-service-config` in collect-data

Reply:
The PR says it is moving the RSC to being managed in infra-deployments instead of here. So, we are still creating a CR per env. Why do we now need to create it in the adapter?

Reply:
removed persistance